packages/core/src/odata-v2/request-builder/function-import-request-builder.spec.ts

```diff
@@ -169,6 +168,10 @@ describe('FunctionImportRequestBuilder', () => {
   });
 
   it('return undefined or throw in failure case', async () => {
+    nock(defaultHost)
+      .get(`${serviceUrl}/TestFunctionImportNoReturnType?$format=json`)
```

Comment:
I would prefer it if we would use the `query` option of nock. This makes it in dependent of the order of the parameters and I find it better to read (see the test above).
`.query({ $format: 'json' })`

Reply:
I saw this in the other spec files as well.